src/Accelerators/NNPA/Dialect/ZHigh/ZHighOps.hpp

```diff
@@ -12,6 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#ifdef __NNPA__
```

Comment:
What file get #defed out? 
Is there not a more elegant way to only include in the makefile subdirectories if we build for the NNPA?
Because if the dialect is not built, then every file using dialect ops must also be #defed out, that seem somewhat ugly.

Or is it because the ZHighOps.hpp must be included in some non-nnpa code? It might be cleaner to #def the include out then.

My recommendation is that no files under NNPA should be #def out, and wherever there are needed references from regular onnx-mlir, that is where the #def should be present.

Reply:
removed